### DISCUSSION TITLE
Bupropion PBPK Model

### DISCUSSION THREAD

**Discussion by Fatima-Marok** [Open-Systems-Pharmacology/Forum#739] [Announcement]
Dear OSP-community,

We have been working on a PBPK model of the CYP2B6 victim drug bupropion and its three metabolites hydroxybupropion, erythrohydrobupropion and threohydrobupropion. The model is able to predict and describe drug-drug(-gene) interactions (DD(G)I) with rifampicin (CYP2B6, CYP2C19 and UGT2B7 perpetrator drug), fluvoxamine (CYP2C19 perpetrator drug) and voriconazole (CYP2B6 and CYP2C19 perpetrator drug).

The corresponding manuscript, including comprehensive documentation, has been published online in Pharmaceutics and the bupropion model as well as PK-Sim projects presenting the predicted DD(G)I studies can be found [here](https://github.com/Open-Systems-Pharmacology?q=ddi+pbpk+osp-model+pk-sim&type=&language=).

Related publication:
Marok FZ, Fuhr LM, Hanke N, Selzer D, Lehr T. [Physiologically Based Pharmacokinetic Modeling of Bupropion and Its Metabolites in a CYP2B6 Drug-Drug-Gene Interaction Network.](https://www.mdpi.com/1999-4923/13/3/331/htm#B59-pharmaceutics-13-00331) Pharmaceutics. 2021; 13(3),331

All the best,
Fatima

**Comment by StephanSchaller** [comment]
Congrats! Great job! :-)

**Comment by teutonicod** [comment]
Hi, 

the links to this publication don't work anymore, could you please update them?
Thanks a lot!

Donato

**Reply by Yuri05** [reply]
The 2nd link is working;
the first link is supposed to be this one I assume: https://github.com/Open-Systems-Pharmacology/Bupropion-Model
